app/assets/stylesheets/components/_result.scss

```diff
@@ -1,149 +1,44 @@
 .result {
   @include clearfix();
-  border: 1px solid $result-border-color;
-  border-radius: $result-radius;
-  margin-bottom: $baseline-unit*6;
 
-  .svg-icon {
-    @include inline-block();
-    position: relative;
-    top: 5px;
-    margin-right: $baseline-unit;
-  }
+  margin-bottom: $baseline-unit*6;
+  padding: $baseline-unit;
 
-  .svg-icon-email,
-  .svg-icon-url,
-  .svg-icon-phone {
-    width: 20px;
-    height: 20px;
-    fill: $color-green-secondary;
-  }
+  border: 1px solid $result-border-color;
+  border-radius: $result-radius;
+  -webkit-box-shadow: 0 1px 0 0 $result-shadow-color;
+  -moz-box-shadow: 0 1px 0 0 $result-shadow-color;
+  box-shadow: 0 1px 0 0 $result-shadow-color;
 }
 
 .result__heading {
-  background-color: $background-color;
+  @include column(12);
   @include body(18, 27);
-  padding: $baseline-unit*2;
-  color: $result-heading-color;
-  margin-top: 0;
-  margin-bottom: 0;
-
-  @include respond-to ($mq-m) {
-    margin-bottom: $baseline-unit*2;
-  }
-}
 
-.result__heading-text {
-  @include respond-to($mq-xs, $mq-m) {
-    display: block;
-    padding-left: 20px;
-  }
-}
-
-.result__adviser-distance {
-  display: block;
-  line-height: 27px;
-  color: $color-black;
-  padding: 0 $baseline-unit*2;
   margin-top: 0;
-  font-weight: 300;
-  padding-left: 20px;
-
-  @include respond-to ($mq-m) {
-    display: inline;
-    margin-top: $baseline-unit*2;
-    padding-left: 0;
-  }
-}
+  margin-bottom: 0;
 
-.result__sub-heading {
-  @include body(16, 24);
   color: $result-heading-color;
-  font-weight: 500;
-  margin-top: 0;
-}
-
-.result__address {
-  @include column(12);
-  margin-top: $baseline-unit*4;
-  margin-bottom: $baseline-unit*4;
 
-  @include respond-to (37.5em) {
-    @include column(4);
-  }
-
-  @include respond-to ($mq-m) {
-    @include column(3);
-  }
-}
-
-.result__advice {
-  @include column(12);
-
-  @include respond-to (37.5em) {
-    margin-top: $baseline-unit*4;
-    @include column(8);
-  }
-
-  @include respond-to ($mq-m) {
-    @include column(9);
-    margin-top: $baseline-unit*4;
-    margin-bottom: $baseline-unit*4;
-    padding-left: $baseline-unit*2;
-    border-left: 1px solid $result-border-color;
-  }
-}
-
-.result__2col {
-  margin-top: $baseline-unit*4;
-
-  @include respond-to (37.5em) {
-    @include column(6);
-    margin-top: 0;
-    margin-bottom: $baseline-unit;
-  }
-}
-
-.result__keyword {
-  margin-top: $baseline-unit*2;
-  float: left;
-
-  @include respond-to ($mq-m) {
-    @include column(12);
+  @include respond-to($mq-m) {
+    padding-top: $baseline-unit*2;
   }
 }
 
-.result__list {
-  @extend .unstyled-list;
+.result__heading-text {
+  font-size: 1.4rem;
```

Comment:
I think there is a preference to use the `body` helper for defining font size and line height.